compiler-cli/src/remove.rs

```diff
@@ -0,0 +1,41 @@
+use std::path::{Path, PathBuf};
+
+use gleam_core::{
+    error::{FileIoAction, FileKind},
+    Error, Result,
+};
+
+use crate::{cli, fs};
+
+pub fn command(packages: Vec<String>, dev: bool) -> Result<()> {
+    // Read gleam.toml so we can remove deps from it
+    let mut toml = fs::read("gleam.toml")?
+        .parse::<toml_edit::Document>()
+        .map_err(|e| Error::FileIo {
+            kind: FileKind::File,
+            action: FileIoAction::Parse,
+            path: PathBuf::from("gleam.toml"),
+            err: Some(e.to_string()),
+        })?;
+
+    // Insert the new deps
+    for package_to_remove in packages {
+        let dependency_key = if dev {
+            "dev-dependencies"
+        } else {
+            "dependencies"
+        };
+
+        #[allow(clippy::indexing_slicing)]
+        let _ = toml[dependency_key]
+            .as_table_mut()
+            .and_then(|deps| deps.remove(&package_to_remove));
+
+        cli::print_removed(&format!("{package_to_remove}"));
+    }
+
+    // Write the updated config
```

Comment:
We'll need to resolve the dependencies again to remove any other packages that are now unused, and to update the manifest.toml file.

Reply:
Since I happen to have this pulled up, I think you can just add the following after you write the config:
```gleam
let paths = crate::project_paths_at_current_directory();
_ = crate::dependencies::download(&paths, cli::Reporter::new(), None, UseManifest::No)?;
```